data_store.rb

```diff
@@ -0,0 +1,36 @@
+require 'json'
+require_relative 'music_album'
+require_relative 'genre'
+
+module DataStore
+  def load_albums
+    if File.exist?('album.json')
+      JSON.parse(File.read('album.json')).map do |album|
+        MusicAlbum.new(album['name'], album['genre'], album['label'], album['publish_date'], album['on_spotify'])
```

Comment:
There is a missing parameter while saving into the json file, causing the data to shift one spot to the left.